src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -1470,6 +1471,18 @@ Limitation: Automatically played feed videos will show up in your watch history.
 
 Limitation: Feed videos will play for less than 1 minute before encountering playback issues."</string>
 
+    <string name="revanced_spoof_signature_title">Spoof app signature</string>
+    <string name="revanced_spoof_signature_summary_on">App signature spoofed.</string>
+    <string name="revanced_spoof_signature_summary_off">App signature not spoofed.</string>
+    <string name="revanced_spoof_signature_user_dialog_message">"App signature will be spoofed to the original YouTube.
+
+This will only work if the following conditions are met:
+
+1. The original YouTube is installed on the device.
+2. GmsCore have permission to run in the background.
```

Comment:
Is this even a thing? My understanding is that removing battery restrictions automatically allows the app to run in the background. Or am I wrong here?

Reply:
> Is this even a thing? My understanding is that removing battery restrictions automatically allows the app to run in the background. Or am I wrong here?

Nope. Lift microG battery restriction ensure a normal buffering, because microG needs to work in realtime to correctly generate requests to Google servers. This is a minimum ```PoToken``` implementation requirement.